### PR TITLE
chore(early-access-feature): add delete

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- need to be able to delete early access feature
- save condition was incorrect and buggy

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by EDsCODE** [comment]
This migration should be fine. There are no items in the table yet

**Comment by liyiy** [comment]
1. Nothing happens when I click on this button (archive feature)
Screen Shot 2023-05-09 at 12 53 19 PM

2. Perpetual load box here when I go from dropdown -> view btn from the table list view
Screen Shot 2023-05-09 at 12 56 12 PM

3. Nothing happens when I click this button (I think this is fixed now with the site app, but would double check again)
Screen Shot 2023-05-09 at 12 55 22 PM

4. I think the delete button does work? but nothing happens when I click on it
Screen Shot 2023-05-09 at 12 57 27 PM

5. After I create a new feature release and click on the site app, it doesn't show up
Screen Shot 2023-05-09 at 1 27 50 PM

**Comment by EDsCODE** [comment]
For 5, does it show up on refresh?

**Comment by EDsCODE** [comment]
1. Removed. Unnecessary buttons for now
2. N/A now
3. Try this again. I think it should work now. It's technically async which might need some adjustment for better UX
4. This should work now too
5. Question above: does it show up on refresh?

**Comment by liyiy** [comment]
> 1. Removed. Unnecessary buttons for now
> 2. N/A now
> 3. Try this again. I think it should work now. It's technically async which might need some adjustment for better UX
> 4. This should work now too
> 5. Question above: does it show up on refresh?

5) Yes, only on hard refresh

everything else looks good, 

questions though: 

1) is it intentional that I can't delete or edit releases that have been promoted?
2) how do users know they have to also install the site app for it to all work?

**Comment by EDsCODE** [comment]
> 1. is it intentional that I can't delete or edit releases that have been promoted?

Ideally yes, if it's easy to delete, it will make the feature flag conditions confusing. Once a feature hits GA, it should be expected that the rollout is 100% and you can't "delete". If anything, I'm going to add the archiving back but I don't think it's a blocker for initial release

> 2. how do users know they have to also install the site app for it to all work?

They don't necessarily. It's one of two options
Screenshot 2023-05-09 at 3 14 32 PM

**Review by liyiy — APPROVED** [review]

File: frontend/src/scenes/early-access-features/earlyAccessFeatureLogic.ts [hunk 1]
```diff
@@ -91,23 +105,58 @@ export const earlyAccessFeatureLogic = kea<earlyAccessFeatureLogicType>([
...
     })),
     afterMount(async ({ props, actions }) => {
         if (props.id !== 'new') {
             await actions.loadEarlyAccessFeature()
         }
-
-        actions.editFeature(props.id === 'new')
+        if (props.id === 'new') {
+            actions.resetEarlyAccessFeature()
+            actions.editFeature
```
Comment (neilkakkar) [inline frontend/src/scenes/early-access-features/earlyAccessFeatureLogic.ts:159]: perhaps you need to call this?

**Comment by neilkakkar** [comment]
whoops, I'll fix the juggling snapshots!

For a future PR: The instructions for getting all previews is out of date 👀 .

Also, I don't think we can merge this this week, because the migration is blocking - need to wait for move this weekend before we can add more fields to the db, unless @danielxnj says otherwise.

**Pull request closed without merging**